File: documentation-examples-java/src/java/dev/typr/foundations/docs/mariadb/BinaryTypes.java
package dev.typr.foundations.docs.mariadb;

import dev.typr.foundations.MariaType;
import dev.typr.foundations.MariaTypes;

@SuppressWarnings("unused")
public class BinaryTypes {
  // start
  MariaType<byte[]> binaryType = MariaTypes.binaryOf(16);
  MariaType<byte[]> varbinaryType = MariaTypes.varbinaryOf(255);
  MariaType<byte[]> blobType = MariaTypes.blob;
  // stop
}

File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/mariadb/BinaryTypes.kt
package dev.typr.foundationskt.docs.mariadb

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class BinaryTypes {
    //start
    val binaryType: MariaType<ByteArray> = MariaTypes.binaryOf(16)
    val varbinaryType: MariaType<ByteArray> = MariaTypes.varbinaryOf(255)
    val blobType: MariaType<ByteArray> = MariaTypes.blob
    //stop
}
